.github/workflows/test.yml

```diff
@@ -0,0 +1,51 @@
+name: test
+
+on:
+  workflow_call:
+    inputs:
+      php:
+        default: "8.3"
+        type: string
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+
+      - name: Setup Node
+        uses: actions/setup-node@v4
```

Comment:
Maybe add an input for the Node version?

Reply:
I'm not too familiar with GitHub actions, but I'll commit a change.